src/graphql/query/Welcome.js

```diff
@@ -0,0 +1,12 @@
+import { WelcomeType } from '../types/index.js';
+
+export default {
+  type: WelcomeType,
+  args: {},
+  resolve(_, __) {
```

Comment:
use (parents, args) convention
Looks easy to understand

Reply:
Ya ya, parents args will be the convention, this was just a trial one and we weren't using those variables anywhere which is why I just placed it there